executor/adapter.go

```diff
@@ -90,7 +90,7 @@ func (a *recordSet) Next() (*ast.Row, error) {
 
 func (a *recordSet) Close() error {
 	err := a.executor.Close()
-	a.stmt.logSlowQuery(a.lastErr == nil)
+	a.stmt.logSlowQuery(nil, a.lastErr == nil)
```

Comment:
We can obtain the txnTS right after line 161 `buildExecutor`, and store it in recordSet